bleach/__init__.py

```diff
@@ -1,14 +1,24 @@
 # -*- coding: utf-8 -*-
 
 from __future__ import unicode_literals
+import warnings
 
 import packaging.version
 
-from bleach.linkifier import (
+# ignore html5lib deprecation warnings to use bleach; we are bleach
+# apply before we import submodules that import html5lib
+warnings.filterwarnings(
+    "ignore",
+    message="html5lib's sanitizer is deprecated",
+    category=DeprecationWarning,
+    module="bleach._vendor.html5lib",
+)
+
+from bleach.linkifier import (  # noqa: E402 module level import not at top of file
```

Comment:
E402 is so annoying. I can't think of a case where it's helped me. Might be worth excluding in the configuration.